src/renderer/shared/util/getTokens.js

```diff
@@ -0,0 +1,20 @@
+import { sortBy, values } from 'lodash';
+import fetch from 'node-fetch';
+
+import { TOKENS_URL, NETWORK_MAP } from 'shared/values/config';
+
+export default async function getTokens(net, details = false) {
+  const networkKey = NETWORK_MAP[net];
+  const response = await fetch(TOKENS_URL);
+  const tokens = values(await response.json());
+
+  // eslint-disable-next-line
+  return sortBy(tokens, 'symbol').map((token) => {
+    const tokenDetails = token.networks[networkKey];
+    if (tokenDetails) {
+      return details
+        ? { symbol: token.symbol, ...tokenDetails }
+        : tokenDetails.hash;
```

Comment:
⚠️ This changes the original behavior of the `getToken` function, and it now results in `undefined` values being a part of the array.

Reply:
Aaah I see 
I used an older branch having that same issue :P 
Took the logic over thinking we had fixed it - not realizing the piece of code was outdated xD 
I originally also used the getTokens but it rendered to be useless for the piece I worked on